src/Files.App/ViewModels/Dialogs/CreateShortcutDialogViewModel.cs

```diff
@@ -75,6 +91,9 @@ public bool IsLocationValid
 			}
 		}
 
+		private bool _isLocationInPath;
```

Comment:
Is the name of this field appropriate? I think a name like `_hasArguments` would be more appropriate.